test/suites/basic.sh

```diff
@@ -697,11 +697,18 @@ EOF
   IPV4_GW="$(lxc network get lxdbr0 ipv4.address | cut -d/ -f1)"
   IPV6_GW="$(lxc network get lxdbr0 ipv6.address | cut -d/ -f1)"
 
+  # Sometimes ping is not installed by cloud-init, so try to install it manually
+  for m in c1 c2 ; do
+    lxc exec micro01 -- lxc exec "${m}" -- apt-get install -y --no-install-recommends iputils-ping
+  done
+
   lxc exec micro01 -- lxc exec c1 -- ping -nc1 -w5 -4 "${IPV4_GW}"
   lxc exec micro01 -- lxc exec c2 -- ping -nc1 -w5 -4 "${IPV4_GW}"
   lxc exec micro01 -- lxc exec c1 -- ping -nc1 -w5 -6 "${IPV6_GW}"
   lxc exec micro01 -- lxc exec c2 -- ping -nc1 -w5 -6 "${IPV6_GW}"
   if [ "${SKIP_VM_LAUNCH}" != "1" ]; then
+    # Sometimes ping is not installed by cloud-init, so try to install it manually
+    lxc exec micro01 -- lxc exec "v1" -- apt-get install -y --no-install-recommends iputils-ping
```

Comment:
As it's now always installed manually, I think we can also get rid of the install instruction in the cloud init file.

Reply:
Fixed.